src/Phinx/Db/Action/AddColumn.php

```diff
@@ -37,10 +37,10 @@ public function __construct(Table $table, Column $column)
      * @param \Phinx\Db\Table\Table $table The table to add the column to
      * @param string $columnName The column name
      * @param mixed $type The column type
-     * @param mixed $options The column options
-     * @return \Phinx\Db\Action\AddColumn
+     * @param array $options The column options
+     * @return self
```

Comment:
phpcs did not like it having the return type annotation matching the type hint. However, I think both might be wrong here given the static binding, not sure how to resolve? replace `@return self` with `@return static` and remove the inline type hint until phinx exclusively uses PHP 8.0? This is the same with other files in the `Action` sub-folder.

Though could maybe just remove the usage of `static` here and use `new AddColumn` and not have this class be extendable given it doesn't use an interface.

Reply:
Return types should never be self/static until we are fully PHP8, just omit them, then the docs here are easy enough.
The sniffer usually should check that no other return type is inside the scope of the method, and then a return type is not needed and superfluous.

Reply:
Not sure I follow on what you want to see for this specific function or elsewhere. For any case where the return docstring is `@return self` or `@return static`, just omit the inline return type altogether?

Reply:
The functional return type, yes. No return types for self or static references, as they can be purely CS checked.

Reply:
The doc return can can be `static` here. `self` isn't correct for `new static`.